### PR TITLE
Finnish translations + minor changes

### PR DESCRIPTION
**Pull request by Jeb4dev**
###  🇫🇮 Finnish translations 🇫🇮
- Added finnish translations to main readme file.
- Added finnish translations to all 15 readme files of external components.
### Minor changes
- Fixed 2 whitespaces in one translation file.
- Reformatted html in one german translation file.
- Fixed table of contents reference to v1 readme syntax.
- added missing `mouse gover connections` from v1 readme. 
  - Reference was in toc, but was not found in readme

**Did you build the project before submitting the PR?**

- [ ] Yes
- [x] No
I tried to build it but did not have pnpm installed so it failed. I don't think it is required to build as I only added translations to readme files.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -161,7 +161,7 @@ const tsParticles = require("tsparticles-engine");
 
 // or
 
-import { tsParticles } from "tsparticles-engine";
+import {tsParticles} from "tsparticles-engine";
```
Comment (matteobruni) [inline README.md]: I prefer having spaces between curly brackets and imported objects

File: README.md [hunk 2]
```diff
@@ -563,7 +569,8 @@ You can find all video tutorials in the website here: <https://particles.js.org/
 
 **particles.json**
 
-You can find some config samples [here](https://github.com/matteobruni/tsparticles/tree/main/websites/particles.js.org/presets) 📖
+You can find some config
+samples [here](https://github.com/matteobruni/tsparticles/tree/main/websites/particles.js.org/presets) 📖
```
Comment (matteobruni) [inline README.md]: This newline is unnecessary

**Review by matteobruni — CHANGES_REQUESTED** [review]
Some minor changes needed, but good job. I don't speak Finnish, I believe you for that 👍

File: README.md [hunk 3]
```diff
@@ -156,13 +156,13 @@ Starting from version 1.12.11 `import` and `require` can be used to import `tsPa
...
-```javascript
-const tsParticles = require("tsparticles-engine");
+  ```javascript
+  const tsParticles = require("tsparticles-engine");
 
 // or
 
-import { tsParticles } from "tsparticles-engine";
-```
+import {tsParticles} from "tsparticles-engine";
```
Comment (matteobruni) [inline README.md]: This line has an incorrect format, the spaces between curly brackets should be kept

File: README.md [hunk 4]
```diff
@@ -156,13 +156,13 @@ Starting from version 1.12.11 `import` and `require` can be used to import `tsPa
 
 Now you can write something like this
 
-```javascript
-const tsParticles = require("tsparticles-engine");
+  ```javascript
```
Comment (matteobruni) [inline README.md]: This section can be kept in the previous format.

File: utils/prettier/prettier-config.json [hunk 5]
```diff
@@ -6,13 +6,13 @@
     {
       "files": "*.ts",
       "options": {
-        "tabWidth": 4
+        "tabWidth": 1
```
Comment (matteobruni) [inline utils/prettier/prettier-config.json]: Please revert this file, The format I chose is fine for me, and it must remain unchanged.

**Review by matteobruni — CHANGES_REQUESTED** [review]
Still some changes needed

File: utils/prettier/prettier-config.json (hunk 5 above)
Comment (Jeb4dev) [reply utils/prettier/prettier-config.json]: Oopsie, was not meant to commit this one, tried to get prettier to work, but wasn't able to. My apologies.

**Comment by Jeb4dev** [comment]
How should I use prettier, I installed it and it run with I believe correct config but still the tests are yelling at me. Could you advice me?

**Comment by matteobruni** [comment]
> How should I use prettier, I installed it and it run with I believe correct config but still the tests are yelling at me. Could you advice me?

You need to build the project, there are some prettier dependencies for common configurations

**Pull request closed without merging**